tests/plotting/conftest.py

```diff
@@ -0,0 +1,13 @@
+import matplotlib.pyplot as plt
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def close_figures():
```

Comment:
This fixture is now automatically used for all tests in the `plotting/` folder? But tests outside are not affected?

Reply:
As far I understand it, yes.

Reply:
Checked by making `close_figures` raise an exception. That makes only plotting tests fail, everything else is unaffected.